index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Wuyep Abdul</title>
+  </head>
+  <body>
+    <nav>
+      <header class="padding-round">John Doe</header>
+      <div class="bars"><img src="./images/bars.png" alt="image" /></div>
+    </nav>
+
+    <section class="headline">
+      <div class="headline-ellipse">
+        <header>
+          <h1>Laura Jong</h1>
+          <h2>Software Developer</h2>
+        </header>
+      </div>
+
+      <div class="intro">
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <hr />
+        <nav class="lists">
+          <div>
+            <ul>
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </div>
+          <div>
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+
+          <div>
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+              <li></li>
+            </ul>
+          </div>
+        </nav>
+        <hr />
+      </div>
+
+      <nav class="social-media">
+        <ul class="social-media-links">
+          <li><img src="./icons/github.png" alt="image" /></li>
+          <li><img src="./icons/linkeedIn.png" alt="image" /></li>
+          <li><img src="./icons/hi.png" alt="image" /></li>
+          <li><img src="./icons/twitter.png" alt="image" /></li>
+          <li><img src="./icons/em.png" alt="image" /></li>
```

Comment:
- [ ] Please let's consider giving some spaces between your icons to make it easier to click on them. see the difference below 👇 
- Your Icons  
![image](https://user-images.githubusercontent.com/55465133/129119262-bc341497-2e00-4e20-8cc8-afcbffcdc681.png)

- The original 
![image](https://user-images.githubusercontent.com/55465133/129119317-74c92661-7531-4ec6-bb12-8d19ec5a433e.png)

Kindly let's fix this issue and stick to the given design 👍